.gitignore

```diff
@@ -14,3 +14,4 @@ coverage
 test/typescript/.idea/*
 test/typescript/*.js
 test/typescript/*.map
+package-lock.json
```

Comment:
Why `package-lock.json` is in `.gitignore`?

> This file is intended to be committed into source repositories, and serves various purposes:
> 
> Describe a single representation of a dependency tree such that teammates, deployments, and continuous integration are guaranteed to install exactly the same dependencies.
> 
> Provide a facility for users to "time-travel" to previous states of node_modules without having to commit the directory itself.
> 
> To facilitate greater visibility of tree changes through readable source control diffs.
> 
> And optimize the installation process by allowing npm to skip repeated metadata resolutions for previously-installed packages.

ref: https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md